static/js/util/util.js

```diff
@@ -359,3 +359,15 @@ export function validateYear(string) {
   }
   return year;
 }
+
+
+export function makeProfileImageUrl(profile) {
+  let imageUrl = `${SETTINGS.edx_base_url}/static/images/profiles/default_120.png`.
+  //replacing multiple "/" with a single forward slash, excluding the ones following the colon
+    replace(/([^:]\/)\/+/g, "$1");
+  if (profile.profile_url_large) {
+    imageUrl = profile.profile_url_large;
+  }
+
+  return imageUrl;
+}
```

Comment:
ah yes indeed it does! what does the `/+/g` do? If I do this:

```js
var url = "https://foo.bar//asdf";
url.replace(/([^:]\/)\//, "$1");
```

it seems to have the same result ("https://foo.bar/asdf") as the pattern above. ahhh regex, so inscrutable sometimes...